roles/dosomething.haproxy/handlers/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+
+- name: restart haproxy
+  service: name=haproxy state=restarted
```

Comment:
Sorry, I meant add `sudo: yes` to every task that needs it, including this handler and all tasks.